_posts/2018-02-16-infrastructure-and-sprint-work.md

```diff
@@ -1,38 +1,41 @@
 ---
 layout: post
 title:  "Infrastructure and Sprint Work: The Great Balancing Act"
-date:   2018-02-15 12:00:00 -0400
+date:   2018-02-16 12:00:00 -0400
 categories: automation infrastructure
 author: Brad Leyden
 ---
 # Wearing Different Hats
 
 Being an automation engineer can involve a number of things, and on a team like
 ours that still has a relatively new and growing automation team, it does involve
-doing a lot of different things. Often, a lot of different things at once.
+doing a lot of different things -- many times, a lot of different things at once.
 
 One thing I often think about with this job is the balance between doing in-sprint
-work with features as they come through the pipeline and doing more 'big picture'
+work with features as they come through the pipeline and doing more "big picture"
 infrastructure work.
 
-I think there is a natural inclination to spend as much time
-as possible working on QA for features in the sprint because it's a more visible
-and immediate way to help the team. No one wants to feel like they're neglecting
-the rest of the team, so when a QA request comes in there is an understandable
-urge to drop what you're doing to help get that feature over the finish line.
+I think there is a natural inclination to spend as much time as possible working
+on QA for features in the sprint because it's a more visible and immediate way to
+help their fellow engineers. No one wants to feel like they're neglecting the team,
```

Comment:
"their" is used without the proper context. "my" or "our" would be the appropriate choice, given that "I" is the subject.